webapp/components/CommentList/CommentList.vue

```diff
@@ -12,7 +12,7 @@
         >
           {{ post.comments.length }}
         </ds-tag>
-        &nbsp; Comments
+        <span class="list-title">{{ $t('common.comment', null, 0) }}</span>
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-24T08:56:39Z" title="Tuesday, September 24th 2019, 10:56:39 am +02:00">Sep 24, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -12,7 +12,7 @@
         >
           {{ post.comments.length }}
         </ds-tag>
-        &nbsp; Comments
+        <span class="list-title">{{ $t('common.comment', null, 0) }}</span>
```


good catch! I'm sure this was an issue somewhere, but I can't find it now...